README.md

```diff
@@ -177,7 +177,7 @@ bin/convert_graph_immutable.sh src/test/resources/test_graph.txt test_graph.dat
 #### Scala 
 Simply add
 ```
-libraryDependencies += "co.teapot" %% "tempest" % "0.12.0"
+libraryDependencies += "co.teapot" %% "tempest" % "0.13.0"
```

Comment:
I think this is possible, but somewhat dangerous. If a user writes codes assuming certain methods are present, and then we change those methods in a new release, the users's code will stop compiling, even though the user hasn't changed anything at all.  I should figure out how most projects manage version numbers.